src/usr/local/www/system.php

```diff
@@ -506,6 +518,34 @@ function is_timezone($elt) {
 	[min => 1, max => 4]
 ))->setHelp('<span class="badge" title="This feature is in BETA">BETA</span>');
 
+$group = new Form_Group('Special Panels Show/Hide');
+
+$group->add(new Form_Checkbox(
+	'dashboardavailablewidgetspanel',
+	null,
+	'Available Widgets',
+	$pconfig['dashboardavailablewidgetspanel']
+	))->setHelp('Show the Available Widgets panel on the Dashboard.');
+
+$group->add(new Form_Checkbox(
+	'systemlogsfilterpanel',
+	null,
+	'Log Filter',
+	$pconfig['systemlogsfilterpanel']
+))->setHelp('Show the Log Filter panel in System Logs.');
+
+$group->add(new Form_Checkbox(
+	'systemlogsmanagelogpanel',
+	null,
+	'Manage Logs',
+	$pconfig['systemlogsmanagelogpanel']
+))->setHelp('Show the Manage Log panel in System Logs.');
+
+$group->setHelp('These options set which special panels to show or hide.  When a panel is hidden a button to show/hide the panel is displayed in the title bar.
+<br /><span class="badge" title="This feature is in BETA">BETA</span>');
+
```

Comment:
'These options allow certain panels to be automatically hidden on page load. A control is provided in the title bar to un-hide the panel.'